### PR TITLE
Fixes indentation on html file to follow style guide. Adds images.

### PR DESCRIPTION
**Pull request by hailed748**
Lots of style changes to follow style guide. Also replaces temp images with my own. Pretty much good to launch now, I will probably spend rest of day working on scaling issues, you may see on more pull from me.

### CODE REVIEW

**Comment by Tetheta** [comment]
Starting review

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -21,189 +19,184 @@ <h1 class="headline-text">Dagmawi Haile</h1>
...
     <div class="projects">
-    <section>
       <h1 class="titles">Projects</h1>
+      <hr id="proj">
       <div class="container">
         <div class="cards">
-          <a class="card">
-            <span class="card-thumbnail thumb1">
-              <span><h4 class="card-topic">Community</h4></span>
+          <a class="card" href="finalProj.md.html" target="_blank" >
```
Comment (Tetheta) [inline portfolio/src/main/webapp/index.html:131]: A little unusual to have .md.html here, some OS's may not like that. Any particular reason for it?

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -21,189 +19,184 @@ <h1 class="headline-text">Dagmawi Haile</h1>
...
-              <h1 class="card-title">EDC IOT Training</h1>
-              <h2 class="card-subtitle">Going back to Ethiopia</h2>
+              <h1 class="card-title">CaringCart</h1>
+              <h2 class="card-subtitle">Shopping during COVID</h2>
               <p class="card-description">
-                  The EDC group went to Ethiopia and collaboirated with 
-                  Wolkiete University to bring electronics resources and
-                  conduct some IoT projects.
+                Four of my classmates and I created an IoT device that can be used
+                by commiunties, especailly the elderly to purchases groceries for
```
Comment (Tetheta) [inline portfolio/src/main/webapp/index.html:144]: nit: typo on communities and especially

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -21,189 +19,184 @@ <h1 class="headline-text">Dagmawi Haile</h1>
...
             <span class="card-summary">
-              <h1 class="card-title">CaringCart</h1>
-              <h2 class="card-subtitle">Shopping during COVID</h2>
+              <h1 class="card-title">EDC IOT Training</h1>
+              <h2 class="card-subtitle">Going back to Ethiopia</h2>
               <p class="card-description">
-                  Four of my classmates and I created an IoT device that can be used
-                  by commiunties, especailly the elderly to purchases groceries for 
-                  one another.
+                The EDC group went to Ethiopia and collaboirated with
```
Comment (Tetheta) [inline portfolio/src/main/webapp/index.html:161]: Nit: typo on collaborated

File: portfolio/src/main/webapp/style.css [hunk 4]
```diff
@@ -294,7 +308,11 @@ html {
     background-color: #FBFBFB;
     padding: 50px 200px;
 }
- 
+
+#proj{
```
Comment (Tetheta) [inline portfolio/src/main/webapp/style.css:312]: nit: space before {

File: portfolio/src/main/webapp/style.css [hunk 5]
```diff
@@ -425,6 +445,77 @@ span {
...
 
+.photography {
+    align-items: center;
+    background-color: #F6F6F6 ;
+    display: block;
+    padding: 0 0 50px 0px;
+    position: relative;
+}
+
+#photo{
```
Comment (Tetheta) [inline portfolio/src/main/webapp/style.css:456]: nit: space before {

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (hailed748) [reply portfolio/src/main/webapp/index.html:131]: It is a link to a final write up for a project I did. It's not hosted anywhere but I can put it on drive if that is better.

**Comment by hailed748** [comment]
If we make changes after we send it live do we have to do anything special or are those changes reflected?

**Comment by Tetheta** [comment]
Whatever you just did now worked

**Pull request merged**